lib/adapters/FileSystem.js

```diff
@@ -112,6 +107,10 @@ class FileSystem extends AbstractAdapter {
 	 * @returns {Promise<module:@ui5/fs.Resource>} Promise resolving to a single resource or null if not found
 	 */
 	_byPath(virPath, options, trace) {
+		if (this.isExcludedPath(virPath)) {
+			return Promise.resolve();
```

Comment:
Shouldn't this explicitly resolve with `null` instead of the implicit `undefined`?
At least that's what the JSDoc says, right?

Reply:
Yes! Done